### PR TITLE
Splashscreen now displays correctly (Fully)

### PR DESCRIPTION
**Pull request by spencerlievens**

### CODE REVIEW

**Comment by spencerlievens** [comment]
screen-shot-2017-02-26-at-13 48 36

File: src/qt/splashscreen.cpp [hunk 1]
```diff
@@ -94,11 +100,8 @@ SplashScreen::SplashScreen(Qt::WindowFlags f, const NetworkStyle *networkStyle)
 
     pixPaint.end();
 
-    // Set window title
-    setWindowTitle(titleText + " " + titleAddText);
-
     // Resize window and move to center of desktop, disallow resizing
-    QRect r(QPoint(), QSize(pixmap.size().width()/devicePixelRatio,pixmap.size().height()/devicePixelRatio));
+    QRect r(QPoint(), pixmap.size());
```
Comment (UdjinM6) [inline src/qt/splashscreen.cpp]: Why dropping `devicePixelRatio` here?

**Comment by UdjinM6** [comment]
Nice one! 👍 
Slightly tested (mac) and I like that it has no bar now, much cleaner look.

Not sure if dropping `devicePixelRatio` was a good idea though.

**Comment by SCDeveloper** [comment]
It does work on Linux and Windows.

**Comment by UdjinM6** [comment]
Hmm... This probably was a bug, actually...

If so,
```
    float devicePixelRatio      = 1.0;
#if QT_VERSION > 0x050100
    devicePixelRatio = ((QGuiApplication*)QCoreApplication::instance())->devicePixelRatio();
#endif
```
should be removed too.

@crowning- can you please give it a gui stress-test? ;)

**Comment by spencerlievens** [comment]
Here is the truth. I fixed this issue in DarkSilk-Core ages ago and saw you are experiencing the same issue, so I did a copy/paste job from our splashscreen.cpp with some amendments for your selection of logo depending on network ;)

**Comment by UdjinM6** [comment]
FYI: bug confirmed 👍 

@spencerlievens Can you please submit a patch to remove redundant code (`devicePixelRatio` is not used anywhere anymore) I mentioned above?

**Comment by spencerlievens** [comment]
I'll have a hunt through the code and remove devicePixelRatio instances ;)

**Pull request merged**